thavalon-server/src/game/messages.rs

```diff
@@ -0,0 +1,71 @@
+//! Asynchronous engine for running THavalon games
+
+use std::collections::HashSet;
+
+use thiserror::Error;
+
+use super::role::Role;
+use super::{Card, MissionNumber, PlayerId, ProposalNumber};
+
+// Game-related messages
+
+/// Something the player tries to do
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub enum Action {
+    Propose { players: HashSet<PlayerId> },
+    Vote { upvote: bool },
+    Play { card: Card },
+    Declare,
+}
+
+/// A message from the game to a player
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub enum Message {
+    /// Error message, usually when a player does something wrong
+    Error(String),
+
+    /// Sends the player their role and information
+    RoleInformation { role: Role, information: String },
```

Comment:
nit: All others have 1 value per line, should this be similar? Or is it preferred to have it on one line unless it goes over long length?